compiler/src/dotty/tools/dotc/core/unpickleScala2/Scala2Unpickler.scala

```diff
@@ -643,7 +644,7 @@ class Scala2Unpickler(bytes: Array[Byte], classRoot: ClassDenotation, moduleClas
     // Need to be careful not to run into cyclic references here (observed when
     // compiling t247.scala). That's why we avoid taking `symbol` of a TypeRef
     // unless names match up.
-    val isBound = (tp: Type) => {
+    val isBound = { (tp: Type) =>
```

Comment:
This is also unusual.

Reply:
Actually no. This is very common, and arguably objectively better, because it matches the style one gets when using a `{}`-based parameter list for lambdas. Like in
````scala
xs.map { (tp: Type) =>
  ...
}
```